lib/vanagon/platform/windows.rb

```diff
@@ -34,6 +34,143 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"
       end
 
+      def add_repository(definition)
+        definition = URI.parse(definition)
```

Comment:
are definitions expected to be unique for windows? if so why make a random string for the name instead of just using part (ideally the unique part) of the definition?